include/nonstd/variant.hpp

```diff
@@ -1394,6 +1517,16 @@ inline Variant visit( Visitor const & vis, Variant const & v )
         case 4: return vis( get<4>( v ) );
```

Comment:
Are there any reasons for not providing more std-convenient implementation of 'visit' method for C++11/C++14?

Reply:
perhaps, lack of knowledge, perseverance, time, prioritization, focus, decentralization...

Reply:
I mean, there wasn't serious blockers for this?

Reply:
I can't remember very well. It may have been lack of knowledge/experience or perceived complexity that led me to not differentiate visitor for C++98 and (C++11 and later).

Reply:
Only one thing here stuns me here: I see no way to implement std-conformance visitor for C++98/03.  I mean explicit specification of visitor's return type for C++98/03 case. And I still can't decide what to do with it.